#!/usr/bin/env python
PACKAGE = "smoreo"
from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()
gen.add("f",    double_t,    0, "Focal length of the camera", 750,  350, 950)
gen.add("cx",    double_t,    0, "cx of the camera", 740,  300, 900)
gen.add("cy",    double_t,    0, "cy of the camera", 560,  200, 800)
gen.add("cone_h",    double_t,    0, "Cone height", 0.38,  0.2, 0.8)
gen.add("camera_h",    double_t,    0, "Camera heigt", 0.861,  0.3, 1.2)
gen.add("cut_off_y",    int_t,    0, "cut_off_y", 1000,  100, 1440)

exit(gen.generate(PACKAGE, "smoreo", "smoreo"))
